src/main/java/org/cbioportal/persistence/util/RedisCacheUtils.java

```diff
@@ -14,7 +14,9 @@
 import java.util.stream.Collectors;
 
 @Component
-@ConditionalOnProperty(name = "persistence.cache_type", havingValue = "redis")
+@ConditionalOnExpression(
+    "#{environment['persistence.cache_type'] == 'redis' or environment['persistence.cache_type_clickhouse'] == 'redis'}"
```

Comment:
I think we should remove this and be enable caching or disable caching for the whole system... going forward

Reply:
agreed. this is so that we can assess performance without cheating with cache.   we need caching on for legacy because otherwise, the product is totally unusable!

Reply:
and at the same time, when we deploy for demonstration purposes, we want initial load of studyview to compete with legacy, i.e. cache ON.